.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,19 @@
-<!--- Thank you for your contribution! Please provide enough information for others to best review your code. -->
+## Testing
```

Comment:
I am wondering how do I know what does this PR accomplishes? 

One could say that this is in the ticket, but often times the ticket states the approval criteria, but sometimes not. I think we might want something here that asserts that the acceptance criteria is somewhere, or to have the acceptance criteria on the PR

Reply:
We discussed that and came to the conclusion that there are two sides of defining what you accomplish in this PR. One is the **what** has changed in the UI and this would go into the testing section, e.g. try to use the jobs overview page and see if the filtering works or look into the performance profiler and verify that there is no more memory leak. 
The other is **why** it has changed and for this you should take a look into the Jira issue. If it is not in a shape where you would feel someone outside may understand the context of the issue you could either bring it in this shape or if that is somehow not possible you can extend this template. It is just a starting point, not an extensive list of everything allowed to be written here :)